Userland/Libraries/LibWebView/URL.cpp

```diff
@@ -90,7 +93,7 @@ Optional<URL::URL> sanitize_url(StringView url, Optional<StringView> search_engi
         }
     }
 
-    auto result = query_public_suffix_list(url);
+    auto result = query_public_suffix_list(url, search_engine);
     if (!result.has_value())
         return format_search_engine();
```

Comment:
`search_engine` is now passed to `query_public_suffix_list`, but it's not actually used (other than to check if it has a value). Maybe `format_search_engine()` could just return input URL if the search engine is not available instead? https://github.com/SerenityOS/serenity/blob/a957e6867ae486d3d3cdf159f4636f127cf9fdd1/Userland/Libraries/LibWebView/URL.cpp#L80-L81

Reply:
That would seem to be a better solution. I have updated the line inside ```format_search_engine()``` that you mentioned, and removed the previous changes. This new change appears to have the exact same effect as the original change.